apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/FileUpload.kt

```diff
@@ -1,19 +1,37 @@
 package com.apollographql.apollo.api
 
-class FileUpload(val mimetype: String, val filePath: String) {
-  override fun equals(other: Any?): Boolean {
```

Comment:
I didn't find a need to implement equals/hashCode on this class. Am I missing something?